iceoryx_posh/include/iceoryx_posh/internal/mepoo/mepoo_segment.inl

```diff
@@ -76,8 +76,8 @@ inline SharedMemoryObjectType MePooSegment<SharedMemoryObjectType, MemoryManager
             .permissions(SEGMENT_PERMISSIONS)
             .create()
             .and_then([this](auto& sharedMemoryObject) {
-                auto maybeSegmentId = iox::rp::BaseRelativePointer::registerPtr(sharedMemoryObject.getBaseAddress(),
-                                                                                sharedMemoryObject.getSizeInBytes());
+                auto maybeSegmentId = iox::rp::UntypedRelativePointer::registerPtr(sharedMemoryObject.getBaseAddress(),
```

Comment:
Here it is still `rp` namespace. This is fine, but should be consistent with the change comment.

Reply:
On the follow-up PR this will be moved into a `::memory` namespace and the `::rp` namespace will be gone.